docs/templates/specs-rs.mdx

```diff
@@ -0,0 +1,90 @@
+## Performance specs
+
+{%if model == 'RS/RS+'%}
+
+[Datasheet for Reach {{model}} [PDF, 554 KB]](http://files.emlid.com/reachrs/Reach-RS-Plus-Datasheet.pdf)
+
+{%endif%}
+
+{%if model == 'RS2'%}
```

Comment:
No need in creating separate if-clauses each time, you can use an "if-else" construction instead